src/web/routes/application/enter-dob/enter-dob.js

```diff
@@ -9,7 +9,8 @@ const pageContent = ({ translate }) => ({
   buttonText: translate('buttons:continue'),
   dayLabel: translate('enterDob.dayLabel'),
   monthLabel: translate('enterDob.monthLabel'),
-  yearLabel: translate('enterDob.yearLabel')
+  yearLabel: translate('enterDob.yearLabel'),
+  whyWeNeedThis: translate('enterDob.whyWeNeedThis')
```

Comment:
Open to suggestions of better names.

Reply:
What about `explanation` or `moreDetail`, I think either would be better.

Reply:
explanation works for me

Reply:
Updated to explanation.